From e561d309955026ba9bf3b710bc8820f76dd2f009 Mon Sep 17 00:00:00 2001
From: Sailaja Bandarupalli <sbandarupalli@habana.ai>
Date: Tue, 18 Jan 2022 12:50:05 +0000
Subject: [PATCH] Revert "[SW-70899] gaudi2: arc: reset CPU_PLL"

This reverts commit 0cbb3f84800bca238bf7c200c56dd7af846faf62.

Maxim is concerned about some un-known bugs getting discovered when CPU
PLL is reset as part of PRSTN reset. Even if PSOC is reset there are
some blocks/parts still active and CPU PLL reset may cause issues

Will add other patch to revert RR/protection bits programming done to
CPU PLL block when fake security is disabled

Change-Id: Ib665c356e3a8a7b45dfd81fa195f6014a1851851
---

diff --git a/arch/gaudi2/arc/common.c b/arch/gaudi2/arc/common.c
index c5bb6c6..e91149a 100644
--- a/arch/gaudi2/arc/common.c
+++ b/arch/gaudi2/arc/common.c
@@ -65,13 +65,6 @@
 	regmap_writel(0x1, mmPSOC_RESET_CONF_PSOC_PRSTN_RST_CFG);
 	regmap_writel(0x3, mmPSOC_RESET_CONF_ARC_PRSTN_RST_CFG);
 
-	/*
-	 * Reset value of PLL_L_PRSTN_RST_CFG is 0xFFFFFFFD, BIT(1) which corresponds to CPU_PLL
-	 * is not getting reset by default. Now ARCs are getting reset as part of Warm/PRSTn
-	 * reset, CPU_PLL should also need to be resetted
-	 */
-	regmap_writel(0xFFFFFFFF, mmPSOC_RESET_CONF_PLL_L_PRSTN_RST_CFG);
-
 	/* Clear masking of PRSTN from SW reset */
 	regmap_writel(0x0, mmPSOC_GLOBAL_CONF_PRSTN_MASK);
 
